app/utils/can.js

```diff
@@ -1,17 +1,23 @@
 import Ember from 'ember';
 
-export default function can(user, scope, permittable){
-  Ember.assert('Must pass a parameter that implements `permitsRole`',
-               !!permittable.permitsRole);
-
-  if (scope === "read" || user.get('verified')) {
-    return user.get('roles').then(function(roles){
-      return Ember.RSVP.all(roles.map((role) => permittable.permitsRole(role, scope)) );
-    }).then((results) => {
-      return results.indexOf(true) > -1;
-    });
-  } else {
+export default function can(user, scope, permittablePromise){
+  if(scope === 'manage' && !user.get('verified')) {
     return Ember.RSVP.resolve(false);
   }
 
+  let permittable;
+
+  return new Ember.RSVP.Promise(function(resolve) {
```

Comment:
This could be slightly simplified via `Ember.RSVP.resolve`:

```
return Ember.RSVP.resolve(permittablePromise)
  .then(function() {
    // current `can` code 
  });
```